src/Controllers/Api/Resources/UserAngelTypeReferenceResource.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Controllers\Api\Resources;
+
+use Engelsystem\Models\AngelType;
+use Engelsystem\Models\User\User;
+use Engelsystem\Models\UserAngelType;
+
+class UserAngelTypeReferenceResource extends BasicResource
+{
+    public function toArray(): array
+    {
+        /** @var User $user */
+        $user = $this->model->pivotRelated;
+        /** @var AngelType $angelType */
+        $angelType = $this->model->pivotParent;
+        /** @var UserAngelType $userAngelType */
+        $userAngelType = $this->model;
+
+        return [
+            'user' => UserResource::toIdentifierArray($user),
+            'confirmed' => !$angelType->restricted
+                || $userAngelType->supporter
```

Comment:
```suggestion
```
supporters can be unconfirmed and I would prefer showing it

Reply:
The state of unconfirmed supporters is imho more a bug thats cahtched here so that handling is ok